test/lit.cfg

```diff
@@ -322,6 +322,30 @@ config.round_trip_syntax_test = make_path(config.swift_utils, 'round-trip-syntax
 config.link = lit.util.which('link', config.environment.get('PATH', '')) or      \
               lit.util.which('lld-link', config.environment.get('PATH', ''))
 
+config.slowest_tests = {
```

Comment:
```suggestion
config.early_tests = {
```

Reply:
Thanks. I'm still actually discussing a refinement to upstream (which converged on `early_tests`). I had wanted to add `late_tests` to let us schedule the "complier crasher" tests late, but now I'm of the opinion that we should just have `test_phases` that is flexible enough to allow minimal maintenance for those that want it. If you want to follow along: https://reviews.llvm.org/D97046